libr/include/r_util/r_assert.h

```diff
@@ -0,0 +1,116 @@
+#ifndef R_ASSERT_H
+#define R_ASSERT_H
+
+#include "r_log.h"
+
+R_API void r_assert_log(RLogLevel level, const char *fmt, ...);
+
+#if defined (__GNUC__) && defined (__cplusplus)
+#define R_FUNCTION     ((const char*) (__PRETTY_FUNCTION__))
+#elif defined (__STDC_VERSION__) && __STDC_VERSION__ >= 199901L
```

Comment:
maybe add `#if __STDC__ ` in the line that checks for `__STDC_VERSION__ >= 199901L` too?

Reply:
ok